config.md
module.exports = {

    // BOT VERSION //
    botVersion: "BETA-v1.1.0",
    
    // BOT INFO //
    prefix: "?", // Default prefix
    status: "dnd",
    eventListeners: 100,
    dev: ".huts.",
    devBy: "| Developed by Huts",
    developers: "980910907695722568",
    noPerms: `You **do not** have the required permissions to use this command!`,
    ownerOnlyCommand: `This command is **only** available for the owner of the bot!`,
    filterMessage: "Your message includes profanity which is **not** allowed!",
    botInvite: "https://discord.com/oauth2/authorize?client_id=1273317561747902515",
    botServerInvite: "https://discord.gg/zFVTE5qkbp",

    // EMBED COLORS //
    embedColor: "Blurple",
    embedAutomod: "Blue",
    embedCommunity: "Green",
    embedModLight: "Red",
    embedModHard: "DarkRed",
    embedInfo: "LuminousVividPink",
    embedMusic: "Gold",
    embedMiniGames: "Orange",
    embedFun: "Yellow",
    embedDev: "Aqua",
    embedProfile: "Navy",
    embedAuditLogs: "Purple",
    embedLevels: "Fuchsia",
    embedEconomy: "DarkOrange",
    embedVerify: "DarkGreen",

    // EMOJIS //
    automodEmoji: "<:auto:1235660206856474704>",
    modEmojiHard: "<a:mod:1235642403986083840>",
    modEmojiLight: "<a:wompus:1235671799241510973>",
    pepeCoffeeEmoji: "<:pepe:1238878395303989309>",
    arrowEmoji: "⤵",
    errorEmoji: "❌",
    auditLogEmoji: "📋",
    verifyEmoji: "<a:ver:1244732033339494450>",
    countSuccessEmoji: "<a:tick:1235674049032486945>",

    // MUSIC EMOJIS //
    musicEmojiPlay: "▶️",
    musicEmojiStop: "⏹️",
    musicEmojiQueue: "📄",
    musicEmojiSuccess: "☑️",
    musicEmojiRepeat: "🔁",
    musicEmojiError: "❌",

    // CHANNEL IDS //
    slashCommandLoggingChannel: "1255219987833163877", // slash command logging channel
    prefixCommandLoggingChannel: "1255219962080133200", // Prefix command logging channel
    suggestionChannel: "1253764481033568277", // Suggestion channel
    bugReportChannel: "1253719805194666114", // Bug report channel
    botLeaveChannel: "1255220259653419008", // Logging channel for bot leaving servers
    botJoinChannel: "1254897835304943717", // Logging channel for bot joining servers
    commandErrorChannel: "1273336191210754110", // Logging channel for command errors
    evalLogsChannel: "1273764268248010863",

    // Ticket Configuration //
    ticketName: 'ticket-',
    ticketDescription: '🗳️ Ticket has been open by',
    ticketCreate: '✅ Your ticket has been created',
    ticketAlreadyExist: 'Sorry but you already have a ticket open. If you want to open a new ticket, please close the current one.',
    ticketNoPermissions: 'Sorry, but you **do not** have permission to do this.',
    ticketError: 'Something went wrong, try again later.',
    ticketMessageTitle: 'Welcome, thanks for opening a ticket. Please describe your problem in detail.',
    ticketMessageDescription: 'A member of our moderation team will soon take care of your request.\nThank you for waiting patiently.',
    ticketMissingPerms: 'Sorry, it looks like I am missing the required permissions to do this. Try giving me higher permissions.',
    ticketClose: 'Close',
    ticketCloseEmoji: '📪',
    ticketLock: 'Lock',
    ticketLockEmoji: '🔒',
    ticketUnlock: 'Unlock',
    ticketUnlockEmoji: '🔓',
    ticketClaim: 'Claim',
    ticketClaimEmoji: '👋',
    ticketManage: 'Members',
    ticketManageEmoji: '➕',
    ticketManageMenuTitle: 'Choose a member.',
    ticketManageMenuEmoji: '❔',
    ticketCloseTitle: 'This ticket is being closed...',
    ticketCloseDescription: 'Ticket will be closed in 5 seconds.',
    ticketSuccessLocked: 'Ticket was locked successfully.',
    ticketAlreadyLocked: 'This ticket is already locked.',
    ticketSuccessUnlocked: 'Ticket was unlocked successfully.',
    ticketAlreadyUnlocked: 'This ticket is already unlocked.',
    ticketSuccessClaim: 'Ticket was successfully claimed by',
    ticketAlreadyClaim: 'Ticket is already claimed by',
    ticketDescriptionClaim: ', it was claimed by',
    ticketTranscriptMember: 'Member:',
    ticketTranscriptTicket: 'Ticket:',
    ticketTranscriptClaimed: 'Claimed:',
    ticketTranscriptModerator: 'Moderator:',
    ticketTranscriptTime: 'Time:',
    ticketMemberAdd: 'has been added to the ticket.',
    ticketMemberRemove: 'has been removed from the ticket.',
}

src/config.js
module.exports = {

    // BOT VERSION //
    botVersion: "BETA-v1.1.0",
    
    // BOT INFO //
    prefix: "?", // Default prefix
    status: "dnd",
    eventListeners: 100,
    dev: ".huts.",
    devBy: "| Developed by Huts",
    developers: "980910907695722568",
    noPerms: `You **do not** have the required permissions to use this command!`,
    ownerOnlyCommand: `This command is **only** available for the owner of the bot!`,
    filterMessage: "Your message includes profanity which is **not** allowed!",
    botInvite: "https://discord.com/oauth2/authorize?client_id=1273317561747902515",
    botServerInvite: "https://discord.gg/zFVTE5qkbp",

    // EMBED COLORS //
    embedColor: "Blurple",
    embedAutomod: "Blue",
    embedCommunity: "Green",
    embedModLight: "Red",
    embedModHard: "DarkRed",
    embedInfo: "LuminousVividPink",
    embedMusic: "Gold",
    embedMiniGames: "Orange",
    embedFun: "Yellow",
    embedDev: "Aqua",
    embedProfile: "Navy",
    embedAuditLogs: "Purple",
    embedLevels: "Fuchsia",
    embedEconomy: "DarkOrange",
    embedVerify: "DarkGreen",

    // EMOJIS //
    automodEmoji: "<:auto:1235660206856474704>",
    modEmojiHard: "<a:mod:1235642403986083840>",
    modEmojiLight: "<a:wompus:1235671799241510973>",
    pepeCoffeeEmoji: "<:pepe:1238878395303989309>",
    arrowEmoji: "⤵",
    errorEmoji: "❌",
    auditLogEmoji: "📋",
    verifyEmoji: "<a:ver:1244732033339494450>",
    countSuccessEmoji: "<a:tick:1235674049032486945>",

    // MUSIC EMOJIS //
    musicEmojiPlay: "▶️",
    musicEmojiStop: "⏹️",
    musicEmojiQueue: "📄",
    musicEmojiSuccess: "☑️",
    musicEmojiRepeat: "🔁",
    musicEmojiError: "❌",

    // CHANNEL IDS //
    slashCommandLoggingChannel: "1255219987833163877", // slash command logging channel
    prefixCommandLoggingChannel: "1255219962080133200", // Prefix command logging channel
    suggestionChannel: "1253764481033568277", // Suggestion channel
    bugReportChannel: "1253719805194666114", // Bug report channel
    botLeaveChannel: "1255220259653419008", // Logging channel for bot leaving servers
    botJoinChannel: "1254897835304943717", // Logging channel for bot joining servers
    commandErrorChannel: "1273336191210754110", // Logging channel for command errors
    evalLogsChannel: "1273733451677306880",

    // Ticket Configuration //
    ticketName: 'ticket-',
    ticketDescription: '🗳️ Ticket has been open by',
    ticketCreate: '✅ Your ticket has been created',
    ticketAlreadyExist: 'Sorry but you already have a ticket open. If you want to open a new ticket, please close the current one.',
    ticketNoPermissions: 'Sorry, but you **do not** have permission to do this.',
    ticketError: 'Something went wrong, try again later.',
    ticketMessageTitle: 'Welcome, thanks for opening a ticket. Please describe your problem in detail.',
    ticketMessageDescription: 'A member of our moderation team will soon take care of your request.\nThank you for waiting patiently.',
    ticketMissingPerms: 'Sorry, it looks like I am missing the required permissions to do this. Try giving me higher permissions.',
    ticketClose: 'Close',
    ticketCloseEmoji: '📪',
    ticketLock: 'Lock',
    ticketLockEmoji: '🔒',
    ticketUnlock: 'Unlock',
    ticketUnlockEmoji: '🔓',
    ticketClaim: 'Claim',
    ticketClaimEmoji: '👋',
    ticketManage: 'Members',
    ticketManageEmoji: '➕',
    ticketManageMenuTitle: 'Choose a member.',
    ticketManageMenuEmoji: '❔',
    ticketCloseTitle: 'This ticket is being closed...',
    ticketCloseDescription: 'Ticket will be closed in 5 seconds.',
    ticketSuccessLocked: 'Ticket was locked successfully.',
    ticketAlreadyLocked: 'This ticket is already locked.',
    ticketSuccessUnlocked: 'Ticket was unlocked successfully.',
    ticketAlreadyUnlocked: 'This ticket is already unlocked.',
    ticketSuccessClaim: 'Ticket was successfully claimed by',
    ticketAlreadyClaim: 'Ticket is already claimed by',
    ticketDescriptionClaim: ', it was claimed by',
    ticketTranscriptMember: 'Member:',
    ticketTranscriptTicket: 'Ticket:',
    ticketTranscriptClaimed: 'Claimed:',
    ticketTranscriptModerator: 'Moderator:',
    ticketTranscriptTime: 'Time:',
    ticketMemberAdd: 'has been added to the ticket.',
    ticketMemberRemove: 'has been removed from the ticket.',
}